.github/workflows/osx.yml

```diff
@@ -69,49 +69,49 @@ jobs:
           mkdir -p kokkos/{build,install}
           cd kokkos/build
           cmake \
-          -DKokkos_ENABLE_${{ matrix.backend }}=ON \
-          -DCMAKE_CXX_FLAGS="-Werror" \
-          -DCMAKE_CXX_STANDARD=17 \
-          -DKokkos_ENABLE_COMPILER_WARNINGS=ON \
-          -DKokkos_ENABLE_DEBUG_BOUNDS_CHECK:BOOL=${{ matrix.debug_bounds_check }} \
-          -DKokkos_ENABLE_DEPRECATED_CODE_3=OFF \
-          -DKokkos_ENABLE_TESTS=OFF \
-          -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
-          -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }} \
-          -DCMAKE_INSTALL_PREFIX=$PWD/../install \
-          ..
+          -S "$PWD/.." \
+          -B "$PWD" \
+          -D CMAKE_CXX_FLAGS="-Werror" \
+          -D CMAKE_CXX_STANDARD=17 \
+          -D CMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }} \
+          -D CMAKE_INSTALL_PREFIX=$PWD/../install \
+          -D Kokkos_ENABLE_${{ matrix.backend }}=ON \
+          -D Kokkos_ENABLE_COMPILER_WARNINGS=ON \
+          -D Kokkos_ENABLE_DEBUG_BOUNDS_CHECK:BOOL=${{ matrix.debug_bounds_check }} \
+          -D Kokkos_ENABLE_DEPRECATED_CODE_3=OFF \
```

Comment:
```suggestion
```
This can be removed as part of cleanup, but not requiring a rerun of CI if you'd rather we defer that to a later and more broad cleanup

Reply:
Okay, will add that with the c++ standard change